tools/topology/topology2/production/tplg-targets-ace3.cmake

```diff
@@ -19,6 +19,11 @@ PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt-sof-ptl-rt721-4ch.bin"
 SDW_AMP_FEEDBACK=false,SDW_SPK_STREAM=Playback-SmartAmp,SDW_DMIC_STREAM=Capture-SmartMic,\
 SDW_JACK_OUT_STREAM=Playback-SimpleJack,SDW_JACK_IN_STREAM=Capture-SimpleJack"
 
+"cavs-sdw\;sof-ptl-rt722-96k\;PLATFORM=ptl,SDW_DMIC=1,NUM_SDW_AMP_LINKS=1,\
+SDW_AMP_FEEDBACK=false,SDW_SPK_STREAM=Playback-SmartAmp,SDW_DMIC_STREAM=Capture-SmartMic,\
+SDW_JACK_OUT_STREAM=Playback-SimpleJack,SDW_JACK_IN_STREAM=Capture-SimpleJack,\
+JACK_RATE=96000,DEEP_BUF_JACK_RATE=96000"
+
```

Comment:
yes production is the right place.

Reply:
> Is production folder correct place for this? I don't think we have any kernel configuration that would use this out-of-the-box...?

Currently, there is no product uses the tplg by default. However, it can be used with `options snd-sof-pci tplg_filename=sof-ptl-rt722-96k.tplg`